pkg/boot/uefi/uefi.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package uefi
+
+import (
+	"bytes"
+	"debug/pe"
+	"encoding/binary"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/u-root/u-root/pkg/acpi"
+	"github.com/u-root/u-root/pkg/boot"
+	"github.com/u-root/u-root/pkg/boot/kexec"
+	"github.com/u-root/u-root/pkg/smbios"
+)
+
+const fvEntryImageOffset int64 = 0xA0
```

Comment:
Is this the size of the firmware volume headers + first ffs? Is it okay to be static? @GanShun what do you think of this?

Reply:
Yes. This is tied to how we build payload. I am okay to move this to a configurable variable. However, I am not sure if it's necessary now.